Issuing using `serverXmlFile` config parm in dev mode; parms with 'alias' don't propagate via mojo executor

Using a config parm like:
`<configFile>src/main/liberty/config/server.xml</configFile>`
leads to this failure in dev mode:

> [ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.0.1:dev (default-cli) on project demo: Unable to execute mojo: Unable to parse configuration of mojo io.openliberty.tools:liberty-maven-plugin:3.0.1:create for parameter configFile: Cannot find 'configFile' in class io.openliberty.tools.maven.server.CreateServerMojo -> [Help 1]

The problem seems to be either a limitation (or bug?) with mojo-executor for parameters defined with an alias.  The alias value cannot be used in the way it's being used, it seems.

So the parameter with alias is defined here:

    public class StartDebugMojoSupport extends BasicSupport {

    @Parameter(alias="configFile", property = "serverXmlFile")
    protected File serverXmlFile;

and DevMojo is using the alias like this:

    elements.add(element(name("configFile"), serverXmlFile.getCanonicalPath()));

A simple change to use the regular (non-alias property name) seems to fix this:

    elements.add(element(name("configFile"), serverXmlFile.getCanonicalPath()));

--------

It suggests there could be other issues to at least watch out for in the code, if not to fix now.

E.g. there is at least one other alias:

    @Parameter(property = "deployPackages", defaultValue = "project", alias = "installAppPackages")
    private String deployPackages;

--- a/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java
+++ b/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java
@@ -791,11 +791,11 @@ public class DevMojo extends StartDebugMojoSupport {
                     elements.add(element(name("stripVersion"), "true"));
                     elements.add(element(name("deployPackages"), "project"));
                     if (serverXmlFile != null) {
-                        elements.add(element(name("configFile"), serverXmlFile.getCanonicalPath()));
+                        elements.add(element(name("serverXmlFile"), serverXmlFile.getCanonicalPath()));
                     }
                 } else if (goal.equals("create")) {
                     if (serverXmlFile != null) {
-                        elements.add(element(name("configFile"), serverXmlFile.getCanonicalPath()));
+                        elements.add(element(name("serverXmlFile"), serverXmlFile.getCanonicalPath()));
                     }
                     if (assemblyArtifact != null) {
                         Element[] featureElems = new Element[4];

